opencensus/trace/ext/pyramid/pyramid_middleware.py

```diff
@@ -121,4 +127,17 @@ def _after_request(self, request, response):
 
 def get_context_header(request):
```

Comment:
Remove this and combine with `get_context_headers`? We can set the default value for `header_names` to `[_PYRAMID_TRACE_HEADER]`.